daemon/algod/api/server/v2/utils.go

```diff
@@ -194,6 +194,16 @@ func computeAppIndexFromTxn(tx node.TxnWithStatus, l LedgerForAPI) *uint64 {
 	return computeCreatableIndexInPayset(tx, blk.BlockHeader.TxnCounter, payset)
 }
 
+// getOptionalRound converts an optional round argument into a basics.Round(0) for "latest"
+// or a non-negative integer.
+// TODO: could change this to parse a string argument and allow param to use string value "latest"
+func getOptionalRound(round *uint64) (basics.Round, error) {
```

Comment:
This interface seems like a funny middle ground.  If we only care about supporting "latest" or a number, I think it's fine to only take numbers, and remove all of the ceremony around an error that never happens.  If we wanted to support of kind of tokens besides "latest" we would need to change the input type and the output type (because using 0 as a special case would be insufficient).  I lean toward guessing there are no other good named rounds, so I'd rather just skip the `error` stuff for now.